docs/features/basic-concepts/genesis-file.md

```diff
@@ -50,4 +53,6 @@ The Genesis file is the basis for the entire network initialization，which cont
   
 ## Gentxs
 
-Gentxs contains the transaction set of creating validators in genesis block. 
\ No newline at end of file
+Gentxs contains the transaction set of creating validators in genesis block. 
+The Cosmos SDK provides robust tools for bootstrapping the identities that will start chain via the gen-tx process. gen-tx or a Genesis Transaction are cryptographically signed transactions that are executed during chain initialization that generate a starting set of validators.
```

Comment:
Why `Cosmos SDK`? How about change to `irishub`

Reply:
fixed